### PR TITLE
Add Letsencrypt command available to generate SSL certificate

### PR DESCRIPTION
**Pull request by lexyan**
Letsencrypt available for Synology, successfully tested on DS214play.
Need to stop Webserver before using command.
I use it like this in this script : 
[renewSSL.sh.txt](https://github.com/SynoCommunity/spksrc/files/67416/renewSSL.sh.txt)

### CODE REVIEW

File: cross/tcp_wrappers/Makefile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+CONFIGURE_TARGET = nop
+COMPILE_TARGET = myCompile
+INSTALL_TARGET = myInstall
+
+include ../../mk/spksrc.cross-cc.mk
+
+.PHONY: myCompile
+myCompile:
+	$(RUN) sed -i -e "s,^extern char \*malloc();,/* & */," scaffold.c
```
Comment (Diaoul) [inline cross/tcp_wrappers/Makefile]: Use a patch instead

File: cross/tcp_wrappers/Makefile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	$(RUN) $(MAKE) REAL_DAEMON_DIR=$(STAGING_INSTALL_PREFIX) PREFIX=$(STAGING_INSTALL_PREFIX) linux
+
+.PHONY: myInstall
+myInstall:
+	@$(RUN) mkdir -p $(STAGING_INSTALL_PREFIX)/sbin/
+	@$(RUN) mkdir -p $(STAGING_INSTALL_PREFIX)/lib/
+	#@$(RUN) mkdir -p $(STAGING_INSTALL_PREFIX)/share/man/man8/
+	#@$(RUN) mkdir -p $(STAGING_INSTALL_PREFIX)/share/man/man5/
+	@$(RUN) mkdir -p $(STAGING_INSTALL_PREFIX)/include/
```
Comment (Diaoul) [inline cross/tcp_wrappers/Makefile]: All theses should be made by the make install below

**Comment by Diaoul** [comment]
Where did you get that patch? It's quite big and seems to remove a lot of stuff.

**Comment by lexyan** [comment]
I get it there http://www.linuxfromscratch.org/blfs/view/6.3/basicnet/tcpwrappers.html

**Comment by lexyan** [comment]
I'll redo it with only the letsencrypt part

**Comment by AustinSaintAubin** [comment]
This issue currently under work at under #2019

**Pull request closed without merging**